README.md

```diff
@@ -80,204 +80,8 @@ spec:
 EOS
 ```
 
-## Example Scenario
+## Example Scenarios
```

Comment:
Given that this is going away, could you add a reasonable Getting Started?

Reply:
Not sure what you mean - what is going away? The entire example?

Reply:
Added text to README file - thx!